staging/src/slime.io/slime/modules/meshregistry/pkg/source/zookeeper/conversion.go

```diff
@@ -466,10 +401,7 @@ func escapeDubboMethods(methods []string, methodApplier func(string)) string {
 		if '0' <= b && b <= '9' {
 			return true
 		}
-		if '_' == b {
```

Comment:
I don't know if there are some lint hints, but I think the original way of writing is more "neat"

Reply:
I think return `b== '_'` is recommend and also "neat".